src/constructs/core/stack.ts

```diff
@@ -6,6 +6,7 @@ export interface GuStackProps extends StackProps {
   // This limits GuStack to supporting a single app.
   // In the future, support for stacks with multiple apps may be required
   app: string;
+  migrated?: boolean;
```

Comment:
Minor: perhaps we want to be more specific (`migratedFromCloudFormation`?) in case there are other migration scenarios in the future?

Reply:
👍